### ISSUE TITLE
sms/mms (conversations plugin)

### ISSUE THREAD

**Issue by drogit** [twilio/OpenVBX#407]
anyone out there using the sms-converstions plugin?  and if you do, does it work with mms?  i can see how it receives (despite not showing in inbox view) and i managed to hack in a send function via notes (https://github.com/jpwalters/OpenVBX-Plugin-SMS-Conversation/issues/9) but  it still has issues.  any updated solutions out there?

**Comment by gegere** [comment]
Yes, I do use the plugin on a daily basis for reservation correspondence. It does not work with MMS at this time.

I'd be happy to help as a side project.

**Comment by chetantrivedi7** [comment]
Hi @drogit .i will help you for resolve this issue . I have some good plugins for openvbx as well . contact me on chetan@xsquaretec.com
Thanks
